test/metabase/api/search_test.clj

```diff
@@ -311,6 +311,19 @@
     (tt/with-temp Table [table {:name table-name}]
       (search-request :rasta :q table-name))))
 
+;; But *archived* tables should not appear in search results
+(let [table-name (tu/random-name)]
+  (expect
+    []
+    (tt/with-temp Table [table {:name table-name}]
```

Comment:
no need to change this now but `with-temp` automatically gives objects random names as needed so you could have done this instead:

```clj
(expect
  []
  (tt/with-temp Table [{table-name :name}]
    (search-request :rasta :q table-name, :archived true))
```

You only need to `let` it outside of the `expect` form if you need to refer to it in both the expected and actual forms